src/schema/suffixes/bold.yaml

```diff
@@ -0,0 +1,4 @@
+---
+name: BOLD
```

Comment:
This table is an odd one out in not having full names. It's fine since all have short descriptions with the full names, but wanted to make sure this was the goal.

Reply:
I just changed it to the full name. Thanks!